pyaerocom/aeroval/experiment_output.py

```diff
@@ -566,7 +566,14 @@ def _create_statistics_json(self):
         stats_info = statistics_defaults
         stats_info.update(extended_statistics)
         if self.cfg.statistics_opts.add_trends:
-            stats_info.update(statistics_trend)
+            if self.cfg.processing_opts.obs_only:
+                obs_statistics_trend = {}
+                for key in statistics_trend.keys():
```

Comment:
You do not need the `.keys()` to iterate over the keys of a dictionary.
It is sufficient to write `for key in statistics_trend: ...`